src/app/component/user/components/profile/profile-service/profile.service.ts

```diff
@@ -35,4 +35,8 @@ export class ProfileService {
     return this.http.get<object>(`${this.backEnd}user/isOnline/${this.userId}/`);
   }
 
+  public getEcoPlaces(): any {
```

Comment:
It is bad to leave the 'any' type, so please, define the type of the function

Reply:
done